docs/reference/6-data-import/getting-started/quick-start.md

```diff
@@ -0,0 +1,143 @@
+
+Log in using the details below:
+
+| Option | Value |
+| --- | --- |
+| Name | Grand Bend 2019 Sample |
+| Agent Type | Manual |
+| Enabled | CHECK |
+| Data Maps |     |
+|     | Choose "Grand Bend 2019..." |
+|     | Click "Add Map" |
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_9-45-36.png)
+
+### Step 8. Upload file to Agent
+
+Once you have created the agent, you must upload the CSV file downloaded earlier ("DI-TEST-2019-GrandBend.csv") because this is a manual agent. On the main Agents page, click the "Upload" icon on the agent you just created. Choose the CSV file and hit upload. Back on the main Agents page, the "Files" column for that agent should now be 1. Now, we are ready to load to the ODS.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_9-49-27.png)
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_9-50-17.png)
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_9-52-11.png)
+
+### Step 9. Run the Agent
+
+Execute Data Import's Transform Load command line task to import and transform CSV data to the StudentAssessment endpoint. This can be done by executing the DataImport.Server.TransformLoad.exe from the TransformLoad folder.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2019-9-26_16-20-27.png)
+
+### Step 10. View Logs
+
+Data Import provides logging facilities for files to process, ingestion log details, and a general application log. Select Configure → **Logs** to view this section and details about the previous import job.
+
+Review job status.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_9-58-26.png)
+
+Click on the Ingestion tab. Review the details below. Click the "+" to open extended information about the API payload for that row.
+
+![](https://edfidocs.blob.core.windows.net/$web/img/reference/data-import/getting-started/image2022-11-9_10-21-23.png)
+
+### Support & Reporting Issues
+
+If you need support while using Data Import, you can log an issue in the [Ed-Fi Tracker](https://tracker.ed-fi.org/projects/EDFI/issues).
+
+> [!INFO]
+> The instructions above provide an overview of Data Import's basic features. Additional documentation, including a User Guide, will be added to provide detailed information on each section of Data Import.
+
+## Contents
+
+Read more about getting started with the Data Import tool:
+
+> [!NOTE]
+> Below are two sample data files used in the "Next Steps" referenced in the walkthrough on this page:
+> \- Sample CSV file: [DI-TEST-2019-GrandBend.csv](https://edfi.atlassian.net/wiki/download/attachments/24119444/DI-TEST-2019-GrandBend.csv?version=1&modificationDate=1576003083757&cacheVersion=1&api=v2)
```

Comment:
Later on, we will want to move this file elsewhere. For right now it is fine to leave it in Confluence. There are a number of other files like that (especially in Certification) that we'll move later.

Confluence export put a slash in front of the dash, which means it will just be rendered as a dash. Do you want to change to an asterisk and have a bullet point?